### PR TITLE
#80 add new layout

### PR DESCRIPTION
**Pull request by Art-mimiron**
#80

### CODE REVIEW

File: src/app/header/header.component.scss [hunk 1]
```diff
@@ -1,27 +1,88 @@
...
+.mat-button{
    border:1px solid gray;
 }
 
-.material-icon {
+.material-icon{
    line-height:none;
 }
+.header{
+   padding: 24px 70px 40px 70px;
```
Comment (dmitryy90) [inline src/app/header/header.component.scss:18]: did you check it in low resolution?

File: src/app/header/header.component.scss [hunk 2]
```diff
@@ -1,27 +1,88 @@
...
+      justify-content: space-between;
+      align-items: center;
+   }
+   &_btns{
+      gap: 40px
+   }
+   &_login{
+      border-radius: 20px;
+      padding: 10px 30px;
+      font-size: 13px;
```
Comment (dmitryy90) [inline src/app/header/header.component.scss:36]: use relative values

File: src/app/header/header.component.ts [hunk 3]
```diff
@@ -20,40 +17,36 @@ enum RoleLinks{
...
   ngOnInit(): void {
-    this.store.dispatch(new CheckAuth());
-    this.role$.subscribe(()=>{
-      this.role = this.store.selectSnapshot<string>(UserRegistrationState.role);
-    })
+    this.store.dispatch(new CheckAuth())
   }
   logout(): void {
-    this.store.dispatch(new Logout());
+    this.store.dispatch(new Logout())
```
Comment (dmitryy90) [inline src/app/header/header.component.ts:47]: i believe semicolon is needed

File: src/app/shared/components/filters-list/city-filter/city-filter.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,57 @@
+// import { ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (dmitryy90) [inline src/app/shared/components/filters-list/city-filter/city-filter.component.spec.ts:1]: why dont you remove this file?

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

File: src/app/header/header.component.ts (hunk 3 above)
Comment (Art-mimiron) [reply src/app/header/header.component.ts:47]: I didn't do it manually, when I've pushed changes to git, it automatically removes this semicolon

File: src/app/header/header.component.scss (hunk 1 above)
Comment (Art-mimiron) [reply src/app/header/header.component.scss:18]: changed to %, also added flex-wrap for low-resolution

File: src/app/header/header.component.scss (hunk 2 above)
Comment (Art-mimiron) [reply src/app/header/header.component.scss:36]: fixed

**Review by dmitryy90 — APPROVED** [review]

**Comment by Art-mimiron** [comment]
merged

**Pull request merged**